### PR TITLE
trivial: recover RocksdbPropertyReporter

### PR DESCRIPTION
**Pull request by msmouse**
## Motivation

https://github.com/aptos-labs/aptos-core/pull/423 removed it accidentally

### Have you read the [Contributing Guidelines on pull requests](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md#pull-requests)?

y

### CODE REVIEW

**Review by davidiw — APPROVED** [review]

File: storage/aptosdb/src/lib.rs [hunk 1]
```diff
@@ -160,6 +169,48 @@ fn update_rocksdb_properties(db: &DB) -> Result<()> {
     Ok(())
 }
 
+#[derive(Debug)]
+struct RocksdbPropertyReporter {
```
Comment (sitalkedia) [inline storage/aptosdb/src/lib.rs:173]: Its interesting that we didn't have any tests failing after this was accidentally removed. Seems like we don't have any relevant test for it? May be let's file a follow up issue for adding some tests?

**Review by sitalkedia — APPROVED** [review]

File: storage/aptosdb/src/lib.rs (hunk 1 above)
Comment (msmouse) [reply storage/aptosdb/src/lib.rs:173]: There's no interface coded for other threads to talk with it. Not sure it's worth adding one just for testing.. This affects only the dashboard anyway.

Comment (msmouse) [reply storage/aptosdb/src/lib.rs:173]: Adding the issue is slower than adding the test :trollface:

**Pull request closed without merging**